jax/_src/interpreters/pxla.py

```diff
@@ -2867,6 +2875,7 @@ def from_hlo(name: str,
                keepalive: Any,
                kept_var_idx: set[int],
                backend: xb.XlaBackend,
+               platforms: Sequence[str],
```

Comment:
You don't need platforms here. You can separate platforms in `MeshComputation`'s compile method right? (just like we do for donated_invars?): https://cs.opensource.google/jax/jax/+/main:jax/_src/interpreters/pxla.py;l=2396

Reply:
Sorry I meant `__init__`

Reply:
I have now implemented this suggestion, but I suspect it may be eventually useful to put `platforms` into the compile args, because if there are multiple platforms then the calling convention and the compilation changes. Right now we only ever use multi-platform lowering with AOT export APIs, which take care of the special compilation needs (here https://cs.opensource.google/jax/jax/+/main:jax/_src/interpreters/pxla.py;l=2882)

After I submit this PR I will look at testing the multi-platform lowering in a JIT setting.

Reply:
Ok, sounds good. Once we use this in compilation, then we can add it to compile_args.